iml-util/util.rs

```diff
@@ -126,6 +126,68 @@ pub mod tokio_utils {
     }
 }
 
+pub mod action_plugins {
+    use futures::{Future, FutureExt};
+    use iml_wire_types::{ActionName, ToJsonValue};
+    use std::{collections::HashMap, fmt::Display, pin::Pin};
+
+    type BoxedFuture =
+        Pin<Box<dyn Future<Output = Result<serde_json::value::Value, String>> + Send>>;
+
+    type Callback = Box<dyn Fn(serde_json::value::Value) -> BoxedFuture + Send + Sync>;
```

Comment:
Could you suggest, what the json contains, for example?
What data may appear here?

Reply:
Sure, I'll add a comment, but it's really anything that can be serialized.

This is basically just for RPC, so this is the incoming data from a remote node.

Reply:
Got it, thanks